contrib/utilities/indent

```diff
@@ -15,20 +15,27 @@
 ## ---------------------------------------------------------------------
 
 #
-# This script indents all source files of deal.II according to our
-# usual code formatting standards. It is used to ensure that our
-# code base looks uniform, as uniformity helps make code easier to
-# read.
+# This script indents all source files of deal.II according to our usual
+# code formatting standards. It is used to ensure that our code base looks
+# uniform, as uniformity helps make code easier to read.
 #
 # While we're already touching every file, this script also makes
-# sure we set permissions correctly
+# sure we set permissions correctly and checks for correct unix-style line
+# endings.
 #
 # The script needs to be executed as
-#   ./contrib/utilities/indent
+#   $ ./contrib/utilities/indent
 # from the top-level directory of the source tree, or via
-#   make indent
+#   $ make indent
 # from a build directory.
```

Comment:
I think this falls in the category of things that are obvious to you and me, but not everyone who is not an experience programmer -- they'll just copy the dollar sign. Would you be willing to omit it again?

Same below.

Reply:
I'll note that my prompt looks differently, and so probably does the prompt of most other people. Dollar signs are really just a convention here.

Reply:
I have removed the $ signs again.

The probability that inexperienced users look at that file is next to zero (They read the documentation and execute it). Further, plain script lacks the syntax hilighting abilities we use in the actual documentation (the stuff under `./doc` where we put everything in nice boxes).

Further, in Gentoo we made the experience that you have to show people what they should do, so we show the full prompt in color (with black background) and output [1].

[1] https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Stage

Reply:
Yes, it's probably a moot point here. I don't really know a particularly good way to mark these things up other than saying "To do this, you need to execute the following command:", i.e., making it clear from the text around it that it is a command.